python/ray/actor.py

```diff
@@ -720,6 +721,31 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         max_task_retries = actor_options["max_task_retries"]
         max_pending_calls = actor_options["max_pending_calls"]
 
+        if scheduling_strategy is None or not isinstance(
+            scheduling_strategy, PlacementGroupSchedulingStrategy
+        ):
+            if placement_group not in ("default", None):
+                warnings.warn(
+                    "placement_group parameter is deprecated. Use scheduling_strategy "
```

Comment:
For these and the other items can we be a bit more descriptive on what the user needs to do? E.g. like to the docs? At the moment it is very vague.